admin/controllers/import.php

```diff
@@ -88,19 +92,20 @@ private function CsvImport($type, $dbname)
 			// search for bom - then it is utf-8
 			$bom = pack('CCC', 0xEF, 0xBB, 0xBF);
 			$fc = fread($handle, 3);
-			$convert = strncmp($fc, $bom, 3) !== 0;
-			if ($convert) {
+			$convert = strncmp($fc, $bom, 3) == 0;
+			if ($convert == false) {
 				// no bom - rewind file
 				fseek($handle, 0);
 			}
 
 			// get fields, on first row of the file
 			$fields = array();
-			if (($data = fgetcsv($handle, 1000, ';')) !== false) {
+			if (($data = fgetcsv($handle, 1000, $separator, $delimiter)) !== false) {
 				$numfields = count($data);
 
 				// convert from ansi to utf-8 if required
-				if ($convert) {
+				if ($convert == true) {
+					$msg .= "<p>BOM not found ... converting to UTF-8</p>\n";
```

Comment:
Why is here no "JText()"?
I added it.

Reply:
@Egnarts94 I have not seen this mod in any of your git branches before today. Was it in your local branch? If not - then I would like to know where it was and how did I miss it.